From 40874545fda602bef440527826a83e6a75525835 Mon Sep 17 00:00:00 2001
From: "Thulasimani,Sivakumar" <sivakumar.thulasimani@intel.com>
Date: Wed, 11 Nov 2015 20:16:04 +0530
Subject: [PATCH 02/19] FOR_UPSTREAM [VPG]: drm/i915: Modify Link BW warn to
 debug msg

Compliance test 7.1.1.2 tests driver's capability to recover
when incorrect link rates are provided by panel. In addition
it checks if this is done within reasonable time or not.
This patch replaces the WARN() with debug message as
repeated calls for the function intel_dp_max_link_bw results
in multiple stack dumps and crossing the timeout value.

Change-Id: I2b43d286184067ff642fe929f6e20e700745adc4
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-30099
Signed-off-by: Sivakumar Thulasimani <sivakumar.thulasimani@intel.com>
Reviewed-on: https://android.intel.com:443/498177
---
 drivers/gpu/drm/i915/intel_dp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dp.c b/drivers/gpu/drm/i915/intel_dp.c
index c3e363e..aa4bf5a 100644
--- a/drivers/gpu/drm/i915/intel_dp.c
+++ b/drivers/gpu/drm/i915/intel_dp.c
@@ -162,8 +162,8 @@ int intel_dp_max_link_bw(struct intel_dp *intel_dp)
 			max_link_bw = DP_LINK_BW_2_7;
 		break;
 	default:
-		WARN(1, "invalid max DP link bw val %x, using 1.62Gbps\n",
-		     max_link_bw);
+		DRM_DEBUG_KMS("Unsupported Max Link bw:%x, using 1.62Gbps\n",
+				max_link_bw);
 		max_link_bw = DP_LINK_BW_1_62;
 		break;
 	}
-- 
1.9.1

